### DISCUSSION TITLE
удоление

### DISCUSSION THREAD

**Discussion by cleveland59** [Elllkere/misterfishdpi#5] [General]
здарова, как ваш майнер удолить? при обновлении версии предыдущую папку хрен снесёшь
![image](https://github.com/user-attachments/assets/46ef97f0-ed9c-4e07-b9eb-91185f48bdee)

**Comment by Elllkere** [comment]
Ты подумал, что это майнер потому что в описание драйвера указан биткоин кошелек для донатов? Это опенсурс драйвер https://github.com/basil00/WinDivert, который использует оригинальный zapret, я здесь ничего не менял, мое здесь только сам exe мистерфиша. Драйвер стопается сам после закрытия мистер фиша, либо в cmd от админа sc stop WinDivert

**Reply by cleveland59** [reply]
да я пошутил про майнер на самом деле) насчёт стопа драйвера проверю попозже,странно что я с незакрытой програамой пытался ее удалить.

**Reply by cleveland59** [reply]
спасибо за ответ
